private-templates-service/client/src/store/templateOwner/reducers.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  OwnerState,
+  OwnerType,
+  GET_OWNER_NAME,
+  GET_OWNER_NAME_SUCCESS,
+  GET_OWNER_NAME_FAILURE,
+  GET_OWNER_PROFILE_PICTURE,
+  GET_OWNER_PROFILE_PICTURE_SUCCESS,
+  GET_OWNER_PROFILE_PICTURE_FAILURE,
+  GetOwnerNameAction,
+  GetOwnerProfilePictureAction,
+} from './types';
+
+const initialState: OwnerState = {
+  owner: undefined,
+  isFetching: false,
+}
+
+const initialOwnerState: OwnerType = {
+  displayName: "",
+}
+
+function owner(owner = initialOwnerState, action: GetOwnerNameAction | GetOwnerProfilePictureAction): OwnerType | undefined {
```

Comment:
Can we rename this function to indicate what it's doing?:)

Reply:
sure!